src/hooks/usePotentialMarkets.tsx

```diff
@@ -24,8 +23,8 @@ export const PotentialMarketsProvider = ({ ...props }) => (
 
 export const usePotentialMarkets = () => useContext(PotentialMarketsContext);
 
-const EXCHANGE_CONFIG_FILE_PATH = '/configs/potentialMarketExchangeConfig.csv';
-const POTENTIAL_MARKETS_FILE_PATH = '/configs/potentialMarketParameters.csv';
+const EXCHANGE_CONFIG_FILE_PATH = '/configs/potentialMarketExchangeConfig.json';
```

Comment:
nit: fetching will work, but we could actually import the file on build as well via:
`import potentialMarketExchangeConfig from '../../public/configs/potentialMarketExchangeConfig.json';`

Reply:
Leaving as a fetch, to allow deployers to leave the `.json`s as optional